riff-raff/app/views/deploy/viewDeploy.scala.html

```diff
@@ -1,12 +1,17 @@
 @(implicit request: Security.AuthenticatedRequest[AnyContent, com.gu.googleauth.UserIdentity], record: deployment.Record, verbose: Boolean = false, stopFlag: Boolean = false)
 @import helper.CSRF
 
-@main(s"Deploy for ${record.buildName}", request, List("auto-refresh", "deploy-verbose", "stop-deploy")) {
+@main(s"Deploy for ${record.buildName}", request, List("auto-refresh", "deploy-verbose", "stop-deploy", "notifications")) {
     <style>
     span.message-verbose {
       display:@if(!verbose) {none} else {list-item};
     }
     </style>
+    <script language="JavaScript">
+        window.riffraff = {
+            buildName: '@record.buildName'
```

Comment:
Rather than set a global variable you could set another data attribute on the element.